src/main/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceImpl.java

```diff
@@ -600,8 +600,11 @@ public AdjudicationSummary getAdjudicationSummary(final Long bookingId, final Lo
     }
 
     @Override
-    public Page<VisitWithVisitors<VisitDetails>> getBookingVisitsWithVisitor(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final String visitType, Pageable pageable) {
-        return bookingService.getBookingVisitsWithVisitor(bookingId, fromDate, toDate, visitType, pageable);
+    public Page<VisitWithVisitors<VisitDetails>> getBookingVisitsWithVisitor(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final String visitType, final Integer page, final Integer size) {
```

Comment:
Does this need an integration test?

Reply:
normally all our endpoints should have an int test, but this doesn't look like a new one so should have one already

Reply:
will check what's there already and see what I can add

Reply:
although obviously if it doesn't then, yes please :grin: